lib/tasks/bus_stops_route_creation.rake

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+namespace :bus_stops_route do
+  task :import => :environment do
+    stop_hash = {}
```

Comment:
No names of variable types in the variable name please

Reply:
![Wasn't me](https://media.giphy.com/media/26tPlN7OSKiuUDp6w/giphy.gif)

Reply:
Totally understood